django/core/mail/backends/smtp.py

```diff
@@ -16,17 +16,44 @@ class EmailBackend(BaseEmailBackend):
     def __init__(self, host=None, port=None, username=None, password=None,
                  use_tls=None, fail_silently=False, use_ssl=None, timeout=None,
                  ssl_keyfile=None, ssl_certfile=None,
+                 ssl_cafile=None, ssl_capath=None, ssl_cadata=None,
                  **kwargs):
         super().__init__(fail_silently=fail_silently)
+
+        # server connection
         self.host = host or settings.EMAIL_HOST
         self.port = port or settings.EMAIL_PORT
+        # user identity
         self.username = settings.EMAIL_HOST_USER if username is None else username
         self.password = settings.EMAIL_HOST_PASSWORD if password is None else password
+        # type of TLS (implicit=ssl, explicit=tls)
         self.use_tls = settings.EMAIL_USE_TLS if use_tls is None else use_tls
         self.use_ssl = settings.EMAIL_USE_SSL if use_ssl is None else use_ssl
+        # client cert
+        self.ssl_keyfile = (
```

Comment:
I would stick with the one line if/else statements. The style guide says, "Don’t limit lines of code to 79 characters if it means the code looks significantly uglier or is harder to read."